thirdeye/thirdeye-pinot/src/main/resources/assets/js/lib/self-service.js

```diff
@@ -364,6 +364,7 @@ function addSelfServiceListeners() {
 
```

Comment:
Move to another branch. Onboarding and enabling Spline Regression on TE should be two separated tasks.